text/0000-Project-Goals-2025h2.md

```diff
@@ -0,0 +1,754 @@
+
+## Flagship goals
+
+This period we have 12 flagship goals, broken out into four themes:
+
+* [Beyond the `&`](#beyond-the-), making it possible to create user-defined smart pointers that are as ergonomic as Rust's built-in references `&`.
+* [Unblocking dormant traits](#unblocking-dormant-traits), extending the core capabilities of Rust's trait system to unblock long-desired features for language interop, lending iteration, and more.
+* [Flexible, fast(er) Rust builds](#flexible-faster-rust-builds), making Rust's builds faster and improving support for specialized build scenarios like embedded usage and sanitizers.
+* [Higher-level Rust](#higher-level-rust), making higher-level usage patterns in Rust easier.
+
+### "Beyond the `&`"
+
+| Goal                                                                         | Point of contact | Team(s) and Champion(s)                           |
+| :--                                                                          | :--          | :--                                               |
+| [Nightly support for Autoreborrow traits](https://rust-lang.github.io/rust-project-goals/2025h2/autoreborrow-traits.html)            | [Aapo Alasuutari][]    | [compiler] ([Oliver Scherer][]), [lang] ([Tyler Mandry][]), [types] |
+| [Design a language feature to solve Field Projections](https://rust-lang.github.io/rust-project-goals/2025h2/field-projections.html) | [Benno Lossin][] | [lang] ([Tyler Mandry][]), [types]                        |
+| [Continue Experimentation with Pin Ergonomics](https://rust-lang.github.io/rust-project-goals/2025h2/pin-ergonomics.html)            | [Frank King][]  | [compiler] ([Oliver Scherer][]), [lang] ([TC][])      |
+
+
+One of Rust's core value propositions is that it's a "library-based language"—libraries can build abstractions that feel built-in to the language even when they're not. Smart pointer types like `Rc` and `Arc` are prime examples, implemented purely in the standard library yet feeling like native language features. However, Rust's built-in reference types (`&T` and `&mut T`) have special capabilities that user-defined smart pointers cannot replicate. This creates a "second-class citizen" problem where custom pointer types can't provide the same ergonomic experience as built-in references.
+
+The "Beyond the `&`" initiative aims `&`'s special capabilities, allowing library authors to create smart pointers that are truly indistinguishable from built-in references in terms of syntax and ergonomics. This will enable more ergonomic smart pointers for use in cross-language interop (e.g., references to objects in other languages like C++ or Python) and for low-level projects like Rust for Linux which use smart pointers to express particular data structures.
```

Comment:
```suggestion
The "Beyond the `&`" initiative aims to share `&`'s special capabilities, allowing library authors to create smart pointers that are truly indistinguishable from built-in references in terms of syntax and ergonomics. This will enable more ergonomic smart pointers for use in cross-language interop (e.g., references to objects in other languages like C++ or Python) and for low-level projects like Rust for Linux which use smart pointers to express particular data structures.
```

Or something like that; seems like there's a missing word here.